FreydCategoriesForCAP/gap/FreydCategory.gi

```diff
@@ -19,57 +19,36 @@ InstallMethod( FreydCategory,
   function( underlying_category )
     local freyd_category, to_be_finalized;
     
-    if not HasIsAdditiveCategory( underlying_category ) then
-        
-        Error( "The given category should be additive" );
-        
+    if not IsValidInputForFreydCategory( underlying_category ) then
+        return false;
     fi;
-    
-    ## for IsCongruentForMorphisms to be correct
-    if not CanCompute( underlying_category, "Lift" ) then
-        
-        Error( "The given category should be able to compute Lift" );
-        
-    fi;
-    
-    if not CanCompute( underlying_category, "SubtractionForMorphisms" ) then
-        
-        Error( "The given category should be able to compute SubtractionForMorphisms" );
-        
-    fi;
-    
-    if not CanCompute( underlying_category, "PreCompose" ) then
-        
-        Error( "The given category should be able to compute PreCompose" );
-        
-    fi;
-    
+
     freyd_category := CreateCapCategory( Concatenation( "Freyd( ", Name( underlying_category ), " )" ) );
-    
+
```

Comment:
Just a nitpick: your editor seems to have deleted the whitespace in the empty lines.

If you don't want to "repair" this, I can also do this later.

Reply:
Strange, this was not even displayed by git on my local system. Indeed took a little effort to remove all these white space alternations. If I missed any (which I hope I did not), then please consider merging this PR and make the changes afterwards.